tests/playwright/src/fixtures/electron-app.ts

```
@@ -31,79 +31,182 @@ export const TIMEOUTS = {
   page: Page;
   navigationBar: NavigationBar;
 }
 
+export interface LaunchOptions {
+  timeout?: number;
+  retries?: number;
+}
+
 export const test = base.extend<ElectronFixtures>({
   // eslint-disable-next-line no-empty-pattern
   electronApp: async ({}, use): Promise<void> => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)